scripts/install.sh

```diff
@@ -230,7 +232,10 @@ while [ "$1" != "" ]; do
                                                     ;;
         --ddbToESLambdaErrorThreshold )             shift
                                                     ddbToESLambdaErrorThreshold=$1
-                                                    ;;                                        
+                                                    ;;
+        --archiveConfig )                           shift
```

Comment:
Should we just expose this through to the serverless command as an env var, and avoid the silliness of defining a command option here?